src/lib/components/wizard.component.html

```diff
@@ -1,35 +1,15 @@
 <aw-wizard-navigation-bar
   [direction]="navBarDirection"
-  *ngIf="navBarLocation == 'top' || navBarLocation == 'left'"
-  [ngClass]="{
-    vertical: navBarLocation == 'left',
-    horizontal: navBarLocation == 'top',
-    small: navBarLayout == 'small',
-    'large-filled': navBarLayout == 'large-filled',
-    'large-filled-symbols': navBarLayout == 'large-filled-symbols',
-    'large-empty': navBarLayout == 'large-empty',
-    'large-empty-symbols': navBarLayout == 'large-empty-symbols'
-  }">
+  *ngIf="navigationBarOrientationClassForTopLeft != ''"
+  class="{{navigationBarOrientationClassForTopLeft}} {{navigationBarLayoutClass}}">
```

Comment:
I don't really like this style of using `*ngIf` and `class`. The reason being that it is quite hard to understand when the element will be shown and which classes are appended under which conditions. The old (current) style makes this information clear.

Reply:
Maybe you're right about the style of using the `*ngIf` and `class`, but I think the old way makes redundant checks every change detection cycle, instead of checking every time for a change of `navBarLayout`, change the value when needed.

If you want to make the code here more readable, you can use enums for `navBarLayout` value